java/src/Ice/src/main/java/com/zeroc/Ice/ObjectPrx.java

```diff
@@ -331,6 +333,15 @@ java.util.concurrent.CompletableFuture<com.zeroc.Ice.Object.Ice_invokeResult> ic
      */
     ObjectPrx ice_locatorCacheTimeout(int newTimeout);
 
+    /**
+     * Returns a proxy that is identical to this proxy, except for the locator cache timeout.
+     *
+     * @param newTimeout The new locator cache timeout (in seconds).
+     * @return The proxy with the new timeout.
+     * @see Locator
+     */
+    ObjectPrx ice_locatorCacheTimeout(Duration newTimeout);
+
     /**
      * Returns a proxy that is identical to this proxy, except for the invocation timeout.
```

Comment:
Here we need to specify the unit

Reply:
We do in the param doc `The new invocation timeout (in milliseconds).`